src/main/data/80_webhooks_type.sql

```diff
@@ -0,0 +1,8 @@
+INSERT INTO webhooks_type (type, template)
```

Comment:
It looks like PR #16 already added `data/78_webhooks_type.sql`, so this `INSERT` can be added to that file instead.
Then the conversion in this PR would be updated to manually insert this data, rather than loading the existing `data/78_webhooks_type.sql` file.

Reply:
Just curious - what advantage does it have over this one ? or this PR will cause any issue ?

Reply:
Ahh I just saw the other comment. So basically move all the data to one file.

Reply:
Yep, technically this won't cause an issue, I just think it's more tidy to keep 1 data file per table.
✨ 

Then it's also easier to find in the source.